_content/news/_posts/2018-11-13-kerstkamp-u12.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title: "Kerststage U12 Basket Lummen"
+date: 2018-11-13 08:00:00
+description: Kerststage voor onze U12-spelers
+cover: /news/img/kerstkamp-u8.jpg
+permalink: /news/2018-11-13-kerstkamp-u12/
+custom_js:
+- camp
+---
+
+TODO
```

Comment:
Ook onze drie U12-ploegen komen aan bod tijdens de kerstvakantie: op zaterdag 29 december kunnen de spelers meedoen aan de stagedag van Basket Lummen.
 <br />
 ## Kerststage U12 Basket Lummen
De stagedag is gericht op .....  (aan te vullen).

 ### Voor wie?
Voor jongens en meisjes uit U12 van Basket Lummen.

 ### Praktisch
 Op 29 december van 14h00 tot 18h00.

 ### Inschrijven
Inschrijven is wel nodig en doe je via het formulier hieronder, ten laatste op ?? december 2018. Je betaalt EUR 5,- ter plaatse.

 Over de andere kerstactiviteiten (FROS-kamp en het Bounce Camp Hans Van Wijn) las je eerder al in dit artikel: [Kerstkampen voor de jeugd](/news/2018-10-19-kerstkampen/).